packages/sdk/src/models/decrypted-keypair.ts

```diff
@@ -0,0 +1,50 @@
+import { Keypair as APIKeypair } from '@meeco/keystore-api-sdk';
+import cryppo from '../services/cryppo-service';
+import RSAPrivateKey from './rsa-private-key';
+import RSAPublicKey from './rsa-public-key';
+import { SymmetricKey } from './symmetric-key';
+
+/**
+ * An asymmetric encryption (public/private) key pair, with the private key decrypted.
+ *
+ * Encryption and decryption methods are defined on the classes {@link RSAPublickey} and {@link RSAPrivatekey} which may
+ * be used if only one key is available.
+ */
+export default class DecryptedKeypair {
+  publicKey: RSAPublicKey;
+  privateKey: RSAPrivateKey;
+
+  /** A new random keypair with default length. */
+  static async new(keyBits = 4096): Promise<DecryptedKeypair> {
```

Comment:
Nit: I probably prefer `generate` or `generateNew` over `new` - `new` sort of implies you can pass in private/public keys to create a new instance of the key pair.

Reply:
Eugh that's a whole 8 more characters! Will change it though. Would it make sense to also have other constructor args here though: `DecryptedKeypair.generate(keyBits, externalIds)`

Reply:
Only if it's useful

> Eugh that's a whole 8 more characters
You've done all the hard work of wiring it up now so it's a simple F2 to change it to the more verbose version :D